### PR TITLE
update translation help in readme to be clearer

### PR DESCRIPTION
**Pull request by timcowlishaw**
## Changes in this PR:

Little change to the README.md to make it clearer how to work with translations (updated makemessages command to suppress line numbers to reduce churn, and added extra notes on removing fuzzy strings)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,64 +290,72 @@ We're using [the built-in django translation module](https://docs.djangoproject.
...
-
-   ```console
-   python manage.py compilemessages
+5. Compile the translations to a binary file (this should also be run inside a `fab sh` shell):
+   ```
+   python ./manage.py compilemessages
    ```
 
-### Changing existing translations
+### Changing existing translations:
```
Comment (jacksonj04) [inline README.md:333]:
```suggestion
### Changing existing translations
```

File: README.md [hunk 2]
```diff
@@ -290,64 +290,72 @@ We're using [the built-in django translation module](https://docs.djangoproject.
...
    msgstr "This is my translation"
    ```
 
-1. Compile the translations to a binary file:
-
-   ```console
-   python manage.py compilemessages
+5. Compile the translations to a binary file (this should also be run inside a `fab sh` shell):
+   ```
+   python ./manage.py compilemessages
```
Comment (jacksonj04) [inline README.md:330]: This should be either `python manage.py` or `./manage.py`, not both. Probably `python manage.py`.

File: README.md [hunk 3]
```diff
@@ -290,64 +290,72 @@ We're using [the built-in django translation module](https://docs.djangoproject.
...
    msgstr "This is the new tranlation text I have edited"
    ```
 
-1. Compile the translations again to make your changes show up:
+4. Compile the translations again to make your changes show up (this should be run inside a `fab sh` shell):
 
-   ```console
-   python manage.py compilemessages
+   ```
+   python ./manage.py compilemessages
```
Comment (jacksonj04) [inline README.md:358]: Another `python ./manage.py instance`

**Review by jacksonj04 — CHANGES_REQUESTED** [review]
Some quick tweaks

**Review by dragon-dxw — APPROVED** [review]

**Pull request merged**